src/main/java/org/apache/sling/servlets/post/impl/SlingPostServlet.java

```diff
@@ -151,6 +152,11 @@
                             "content to the repository. By default this is \"j_.*\" thus ignoring all "+
                             "request parameters starting with j_ such as j_username.")
         String servlet_post_ignorePattern() default "j_.*";
+
+        @AttributeDefinition(name="Backwards compatible statuscode",
+                    description="In backwards compatibility mode exceptions will always create a statuscode "
+                        + "500 (see SLING-9896)")
+        boolean backwards_compatible_statuscode() default false;
```

Comment:
I would reformulate it as 'legacy_statuscode_on_persistence_exception`; maybe we would make multiple changes that that need backwards compatibility in the status code.

Reply:
done